### ISSUE TITLE
Select: No aria-label support for Select component

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#3359]
**Describe the bug**
No aria-label support for Select component

**Isolated Example**
https://6pdpt.csb.app/

**To Reproduce**
1. Go to https://6pdpt.csb.app/
2. Run chrome axe DevTool plugin 

**Expected behavior**
No A11y issues

**Screenshots**
![image](https://user-images.githubusercontent.com/5724592/120090769-94dff780-c137-11eb-8fb6-f01908c51e82.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:  0.31.6
`@ui5/webcomponents-react` version: 0.131.4
Operating System:
Browser: Max OS

**Additional context**
I am from SuccessFactors

**Comment by dobrinyonkov** [comment]
Hello @sleric1024,

you should be able to set `aria-label` attribute to the custom element `ui5-select` and the framework will take care to proxy it to the focusable DOM element of the component.

![image](https://user-images.githubusercontent.com/22766569/120152126-c61efd00-c1f5-11eb-948a-35bf1843c951.png)

Kind Regards,
Dobrin

**Comment by sleric1024** [comment]
Hi @dobrinyonkov,
Thanks for your information.
I will close this ticket.
Best Regards,
Eric
